### ISSUE TITLE
MSG_MINIMAL not working

### ISSUE THREAD

**Issue by Jawadgee** [rtCamp/action-slack-notify#45]
I have set `MSG_MINIMAL: true` but I still have 

Ref
Event
Actions URL

I also tried `MSG_MINIMAL: "true"` but result is same

can you help me

**Comment by mrrobot47** [comment]
@Jawadgee which version of action are you using?
Please share your worflow file.

**Comment by Jawadgee** [comment]
Here is the code which sends the notification, 
Also is `MSG_MINIMAL` a Boolean or String

```
      - name: Notification on success
        if: ${{ success() }}
        uses: rtCamp/action-slack-notify@v2.0.2
        env:
          SLACK_TITLE: App deployed
          MSG_MINIMAL: true

```

**Comment by mrrobot47** [comment]
`MSG_MINIMAL` was released in the [last release `v2.1.0`](https://github.com/rtCamp/action-slack-notify/releases/tag/v2.1.0), please use that tag instead of `v2.0.2` and it will work.
